recipes/htsinfer/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 
 source:
   url: https://github.com/zavolanlab/htsinfer/archive/refs/tags/v0.10.0.tar.gz
-  sha256: a851d00a47ef5a47e9c2540599ff2c16723dfac95c12ac85f7152b0ad4c295a7
+  sha256: 5f2f908bc97a97dac611e78e0aa639aa1f372f04e8bf3b11a6887388672a8222
```

Comment:
How did the SHA change without using a new Git tag/version ?!

Reply:
Updated version and SHA, sorry for the oversight